### DISCUSSION TITLE
/is biome in Nether

### DISCUSSION THREAD

**Discussion by TheMartyCZ** [BG-Software-LLC/SuperiorSkyblock2#1444] [Pending Feature Requests] [answered]
Option to have your own menu, to change biomes purely in the nether world.

**Comment by TheMartyCZ** [comment]
How it looks? @OmerBenGera

**Comment by Bragos24** [comment]
Hi, is it possible to add a new feature that allows players to change the biome of the Nether Island? Thanks in advance

**Comment by Lemvs** [comment, marked as answer]
Implemented in latest dev build.

@OmerBenGera you can mark this as answered and close it.
